src/features/quiz/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const preloadImages = [
+  'O버튼.svg',
+  'X버튼.svg',
+  'O버튼-선택.svg',
+  'X버튼-선택.svg',
+  '정답모달.svg',
+  '오답모달.svg',
+  '객관식-코코.svg',
+  '과일바구니.svg',
+  '단답형이미지1.svg',
+  '단답형이미지2.svg',
+  '레벨1코코.svg',
+  '과일바구니-아이템.svg',
+];
```

Comment:
```suggestion
export const PRELOAD_IMAGES= [
  'O버튼.svg',
  'X버튼.svg',
  'O버튼-선택.svg',
  'X버튼-선택.svg',
  '정답모달.svg',
  '오답모달.svg',
  '객관식-코코.svg',
  '과일바구니.svg',
  '단답형이미지1.svg',
  '단답형이미지2.svg',
  '레벨1코코.svg',
  '과일바구니-아이템.svg',
];
```
상수라는 걸 표현하기 위해 스네이크 케이스로 작성해보는 건 어떨까용?

Reply:
좋은 것 같습니다
이런 완전 상수들은 스네이크 케이스가 좋을 듯 하네요👍👍

Reply:
[2793b64](https://github.com/modern-agile-team/8term-coko-Front/pull/98/commits/2793b64892de8b2fb7cbb25a1c097fb6a8e55a99) 커밋에서 수정하였습니다.